src/fava/util/date.py

```diff
@@ -163,9 +204,23 @@ def get_prev_interval(
         return datetime.date(date.year, 1, 1)
     if interval is Interval.MONTH:
         return datetime.date(date.year, date.month, 1)
+    if interval is Interval.FORTNIGHT:
+        year, week, _ = date.isocalendar()
+        w1 = week - 1 if week % 2 == 0 else week
+        if w1 > 2:
+            return datetime.date.fromisocalendar(year, w1, 1)
+        # If we are in the first fortnight of the year, finding the last
+        # fortnight of the previous year is a bit tricky due to being either
+        # starting on week 51 or 53 (if the year has 53 weeks).
+        try:
```

Comment:
This file had complete code coverage in tests (except for the annotated lines), with your changes some cases don't seem to be fully covered - e.g. this try-except-block. Can you add some more tests for those? (`make test` will show the lines or you can take a look at the HTML coverage in `htmlcov`)